backend/eventloop/application_event_loop/application_event_runner_syncruns.go

```diff
@@ -358,6 +358,11 @@ func (a *applicationEventLoopRunner_Action) handleDeletedGitOpsDeplSyncRunEvent(
 	log := a.log
 	log.Info("Received GitOpsDeploymentSyncRun event for a GitOpsDeploymentSyncRun resource that no longer exists")
 
+	if syncOperation.Application_id == "" {
+		log.Info("Application row not found for SyncOperation", "syncOperationID", syncOperation.SyncOperation_id, "applicationID", syncOperation.Application_id)
```

Comment:
Something like this might help us determine this is not an error, next time we see this message in the logs:
"Application row not found for SyncOperation: this is normally because the Application has already been deleted."